cran/repodb.go

```diff
@@ -31,10 +31,15 @@ func NewRepoDb(url RepoURL, dst SourceType, rc RepoConfig, rv RVersion) (*RepoDb
 		repoDatabasePointer.DefaultSourceType = rc.DefaultSourceType
 	}
 
-	if SupportsCranBinary() {
+	if SupportsCranBinary() && rc.RepoType == MPN {
```

Comment:
as discussed on slack - this causes mac/windows binaries to not acquire binaries - see #272 or #273 for alternative implementation(s)

Reply:
Let's go with #273. Conflict is resolved (and we keep the rversion and /contrib in another patch in this branch)